### PR TITLE
ref(whats-new): Convert class to functional component & use named export

### PR DESCRIPTION
**Pull request by priscilawebdev**
Convert the "What's New" sidebar from class to a functional component & other minor things.

Contributes to https://github.com/getsentry/frontend-tsc/issues/2

### CODE REVIEW

File: static/app/components/sidebar/broadcasts.tsx [hunk 1]
```diff
@@ -10,194 +8,117 @@ import SidebarPanel from 'sentry/components/sidebar/sidebarPanel';
...
     }
 
-    this.startPolling();
-  };
-
-  /**
-   * If tab/window loses visibility (note: this is different than focus), stop
-   * polling for broadcasts data, otherwise, if it gains visibility, start
-   * polling again.
-   */
```
Comment (priscilawebdev) [inline static/app/components/sidebar/broadcasts.tsx:134]: if this is `true` the tab is closed and `currentPanel` will have another value.

File: static/app/components/sidebar/broadcasts.tsx [hunk 2]
```diff
@@ -10,194 +8,117 @@ import SidebarPanel from 'sentry/components/sidebar/sidebarPanel';
...
-import withApi from 'sentry/utils/withApi';
+import {useApiQuery, useMutation} from 'sentry/utils/queryClient';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePrevious from 'sentry/utils/usePrevious';
 
 import type {CommonSidebarProps} from './types';
 import {SidebarPanelKey} from './types';
 
-const MARK_SEEN_DELAY = 1000;
```
Comment (priscilawebdev) [inline static/app/components/sidebar/broadcasts.tsx:20]: I don't think this is necessary... we can just send the `PUT` request

File: static/app/components/sidebar/broadcasts.tsx [hunk 3]
```diff
@@ -10,194 +8,112 @@ import SidebarPanel from 'sentry/components/sidebar/sidebarPanel';
...
     }
 
-    this.startPolling();
-  };
-
-  /**
-   * If tab/window loses visibility (note: this is different than focus), stop
-   * polling for broadcasts data, otherwise, if it gains visibility, start
-   * polling again.
-   */
```
Comment (priscilawebdev) [inline static/app/components/sidebar/broadcasts.tsx:134]: by default React Query does not automatically refetch when a tab or window loses visibility

Comment (scttcper) [reply static/app/components/sidebar/broadcasts.tsx:134]: we have that turned off in useApiQuery, you might have to explicitly re-enable it.

**Comment by priscilawebdev** [comment]
I am checking tests

**Review by scttcper — APPROVED** [review]

**Pull request merged**